keychain/models/keychain.py

```diff
@@ -187,11 +193,14 @@ def _get_keys(envs):
                 if key and len(key) > 0  # remove False values
             ]
 
-        if force_env:
```

Comment:
I think this should be put in _retrieve_env() instead

Reply:
How would that work? `_retrieve_env()` only returns the environment name so that it can be used when looking in the main config. This code ignores all that, since `serv_config` already has the correct environment loaded.

Reply:
It's been a while I didn't use server_env, 
So I guess you have something like this : 

```
[prod]
keychain_key: 
   - prod: aabbcc...
   - all: eeffgg..
[dev]
keychain_key:
    - dev: xxyyzz... 
    - all: eeffgg...
```
and `server_config.get(keychain_key)` will return ` ((prod: aabbcc...), (all: eeffgg...))`

So `_retrieve_env()` should  return `('prod', 'all')`
And `config.get(keychain_key_prod)` should return `'aabbcc...'`

Reply:
Why would you have a `prod:` key inside the prod configuration? And what's the purpose of `all:`?

The way I assume people would use it would be to have something like:

  [prod]
  keychain_key: rrttggcc..

  [dev]
  keychain_key: ooeeaa...

Reply:
You can have multiple key for one environment.
Like `prod` to encode stmp credentials and `all` to encode shipping carriers credentials. 
In that way, you ensure mails will not go out in dev and preprod env. But you can ship in all envs (usually carriers won't charge if they don't scan the parcel).

Reply:
Thanks, I think that makes sense. I've updated the code to allow for setting multiple keys in the same environment. `server_environment` actually uses different files, not sections, for each environment, so the format would now be:

  # File dev.conf
  [keychain]
  dev = eerrttee..
  all = oowerrww..

  # File prod.conf
  [keychain]
  prod = qqweertt..
  all = oowerrww..